### ISSUE TITLE
Google, reply please.

### ISSUE THREAD

**Issue by PlebAbhorsen** [Icesythe7/BoL#3]
Hey Google, 

Its Abhorsen from PlebLeaks. I hope you remember me, I have been trying to figure out how to crack the SSL portion of scripts and you told me you would see if you could have TehHelado write a public version of the tool you were using in your skype meeting where I was watching you. Here is a screenshot of a file I happened to come across. :)  Message me your skype/email/something I can use to contact you. #BIGHYPEBOIZ, looks like I will be joining the script cracking team soon.

Talk to you soon mate.

http://i.imgur.com/0A9jvAn.png

**Comment by Icesythe7** [comment]
yes thats my file that was leaked out when my website was cracked

**Comment by PlebAbhorsen** [comment]
? It was leaked when your site was hacked? I obtained that two days ago from other means.

**Comment by PlebAbhorsen** [comment]
And does this mean you will need to rewrite a new script to crack SSL3 since your method is public now if it was leaked during the hack?

**Comment by Icesythe7** [comment]
they don't use ssl3 anymore so it doesn't matter :)

**Comment by PlebAbhorsen** [comment]
?

Wtf do they use now?

**Comment by Icesythe7** [comment]
obfuscation

**Comment by PlebAbhorsen** [comment]
Just write a deobfuscator no? Is that possible?

**Comment by Icesythe7** [comment]
yes but not easy

**Comment by PlebAbhorsen** [comment]
Dude and that Vietnamese homie I was telling you about , told me its possible to auth scripts to your own auth server, even for complicated ones like Evadee, Forbidden Script, SAC, etc. He said the scripts don't check your host file. He even authed me to his Forbidden auth so I can test it and it worked perfectly. Since you don't have to wait for the script to be cracked, you get the legit updates.

**Comment by PlebAbhorsen** [comment]
I thought that wasn't possible until today...

**Comment by PlebAbhorsen** [comment]
Add me on Skype so we can talk easier. Chris.martinez681

**Comment by Icesythe7** [comment]
I know vietnamese homie will be joining plebleaks, we have alot of the
auths on our server already :)

**Comment by PlebAbhorsen** [comment]
Hoat from bots****.com?

**Comment by portes420** [comment]
I need funhouse cracked
